src/app/pages/homepage/homepage.html

```diff
@@ -55,6 +55,23 @@ <h2>Optimized for Angular</h2>
       <p>Built by the Angular team to integrate seamlessly with Angular.</p>
     </div>
   </div>
+
+  <div style="width: 75%">
+  <app-carousel [itemWidth]="250" [itemHeight]="250">
```

Comment:
I don't think the carousel needs to enforce a height on the content, it can just expand to fit them